Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -3835,6 +3835,8 @@ - (void)updateGroupModelTo:(TSGroupModel *)newGroupModel successCompletion:(void
         [message updateWithCustomMessage:updateGroupInfo transaction:transaction];
     }];
 
+    [groupThread fireAvatarChangedNotification];
```

Comment:
`AvatarImageView` wasn't updating because avatar updates in conversation settings don't use `updateAvatarWithAttachmentStream`.  This isn't a perfect solution.  We could also fire the notification anytime a group thread is saved, period.